docs/diagrams/AddLabCommandActivityDiagram.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+
+start
+:User executes add lab command with a Lab Number;
+if () then ([valid Lab Number])
+  if () then ([duplicate lab])
+    :Displays error message;
+  else
```

Comment:
Not sure if this is a compulsory requirement, but maybe this should be `else ([else])` so that `else` will appear on the branch that isn't `[duplicate lab]` in the activity diagram

Reply:
UML notation allows for the else to be left out but you're right now that i think about it, it's best to put it in in case someone wants to nitpick during PE. Thanks!